src/rules/checkParamNames.js

```diff
@@ -245,10 +247,20 @@ const validateParameterNames = (
         return item;
       }).filter((item) => {
         return item !== 'this';
-      }).join(', ');
+      });
+
+      // When disableMissingParamChecks is true tag names can be omitted.
+      // Report when the tag names do not match the expected names or they are used out of order.
+      if (disableMissingParamChecks) {
+        const usedExpectedNames = expectedNames.map(a => a?.toString()).filter(expectedName => !!expectedName && actualNames.includes(expectedName));
```

Comment:
```suggestion
        const usedExpectedNames = expectedNames.map(a => a?.toString()).filter(expectedName => Boolean(expectedName) && actualNames.includes(expectedName));
```

Just prefer to keep it more semantic...